src/compiler/build/test/build-stats.spec.ts

```diff
@@ -18,16 +18,20 @@ describe('generateBuildStats', () => {
   it('should return a structured json object', async () => {
     buildCtx.buildResults = generateBuildResults(config, compilerCtx, buildCtx);
 
-    const result = generateBuildStats(config, buildCtx);
+    const result: d.CompilerBuildStats = generateBuildStats(config, buildCtx);
 
     if (result.hasOwnProperty('timestamp')) {
       delete result.timestamp;
     }
 
+    if (result.hasOwnProperty('compiler') && result.compiler.hasOwnProperty('version')) {
+      delete result.compiler.version;
+    }
```

Comment:
What value was this producing that we had to delete it and remove it from the assertion below?

Reply:
The version that's built at the moment of the test's build (it appended the date and time to the version)

Reply:
Hmmm, what was it that we changed here that made this suddenly fail?

Reply:
It had been failing all along, actually, I just hadn't realized it because I didn't look into the build. My local build version wasn't failing because the timestamp hadn't changed.

Reply:
I guess more concretely, what did we change in this PR to cause this to fail? Or are you saying it was failing since we turned tests back on in CI?

Reply:
I mean quite literally that all the tests in all of my previous commits failed on CI because the built compiler's version includes a timestamp, so I deleted the version that includes the timestamp.

Reply:
Gotcha. Arguably, we should mock timestamp out, but I think that may be out of scope for this PR. Would you mind creating a ticket and adding it to the backlog?

Reply:
STENCIL-137